src/rust/ide/view/graph-editor/src/component/node/action_bar.rs

```diff
@@ -19,9 +19,9 @@ use ensogl_theme as theme;
 // === Constants  ===
 // ==================
 
-const BUTTON_PADDING : f32 = 0.5;
-const BUTTON_OFFSET  : f32 = 0.5;
-
+const BUTTON_PADDING  : f32 = 0.5;
+const BUTTON_OFFSET   : f32 = 0.5;
+const HOVER_PADDING_X : f32 = 15.0;
```

Comment:
I dont understand the nae of the variable. What does it do? Please add a doc or make the name more descripitve.